### ISSUE TITLE
[Idea] - Add the option to move the taskbar and choose the icons to be displayed on desktop in First Steps

### ISSUE THREAD

**Issue by haggen88** [linuxmint/mintwelcome#58]
The LM desktop interface is traditional [very similar to Windows], perhaps it would be useful to have a simple and quick way to configure the desktop with other layouts.  This could facilitate usability for users coming from Mac or other GNU/Linux desktop environments.

For example, Zorin allows you to choose between many layouts. I mean something simpler for Cinnamon.

![zorin appearance](https://user-images.githubusercontent.com/68134588/204114997-b907322c-a5d2-4fe6-942d-109eaf16e0ac.png)

**Comment by fabv12610** [comment]
I agree with what you say, however those settings are in the system settings which is already shown when clicking on "Let's Go'.  However the method to reach the customization of the layout is quite unintuitive
